docs/annexes/SPDX-license-expressions.md

```diff
@@ -191,7 +190,7 @@ A conjunctive license can be expressed in RDF via a `<spdx:ConjunctiveLicenseSet
 
 ```text
 <spdx:ConjunctiveLicenseSet>
-    <spdx:member rdf:resource="http://spdx.org/licenses/GPL-2.0-only"/>
+    <spdx:member rdf:resource="https://spdx.org/licenses/GPL-2.0-only"/>
```

Comment:
This actually needs to stay "http" not "https" since the RDF URI specifies it that way.  Adding the "s" would break compatibility with some RDF oriented tools.